FW/Presentations/Lab_Guides/_C_01_Build_Setup_Download_EDK_II_Win_LabGuide.md

```diff
@@ -0,0 +1,350 @@
+   - Do not use any of the other "... Command Prompts"
+
+
+---
+## Slide 6  Pin VS CMD Prompt 03
+### Pin  VS Command Prompt
+
+
+6. Left Click on "Pin to taskbar"
+
+
+
+---
+## Slide 7  Pin VS CMD Prompt 04
+### Pin  VS Command Prompt
+
+7. Open VS Command Prompt
+
+All Windows Labs use this short-cut to Build Edk II platforms and projects using Windows Visual Studio: 2015/2017 or 2019
+
+
+---
+## Slide 8  Prerequisite Download / Install Tools   
+<br>
+
+---
+## Slide 9  Prerequisites
+### Done Before Class
+
+- Windows 10:
+   - Continuous Integration (CI) - Stuart CI Build with  Visual Studio VS2017 or VS2019
+   - Non Stuart CI - Visual Studio VS2015, VS2017 or VS2019
+- Python 3.8.x or greater and /Scripts directories on Path: [Link](https://www.python.org/) to download 
+- Git for Windows on Path: [Link](http://git-scm.com/download/win)
+- NASM  for Win64: [Link](https://www.nasm.us/pub/nasm/releasebuilds/2.12.02/win64/)
+- IASL Compiler Install iasl from [Link](https://acpica.org/node/199) (iasl-win-20220331.zip)
+
+
+---
+## Slide 10 Setup Simics Environment
+
+Download and Install the Simics Simulator (both Package Manager & Simics-6 Packages)[Simics® Simulator Public Release](https://www.intel.com/content/www/us/en/developer/articles/tool/simics-simulator.html)
+
+Setup the Simics Simulator
+
+[Simics® Simulator Installation and Get Started](https://www.intel.com/content/www/us/en/developer/articles/guide/simics-simulator-installation.html)
+
+Open the "Intel Simics Package Manager"
+
+1. Using the Start menu in Windows, Left Click on "Windows Key" Lower Left
+2. Scroll down from the scroll bar on the right until "Intel Simics Package Manager"
+
+Here is a snapshot of Intel Simics Package Manager with "My Simics Project 1" created (see powerpoint for picture)
+
+
+---
+## Slide 11  Download EDK II Open Source Repos
+
+Use Git Bash to download EDK II
+
+
+---
+## Slide 12  Create Workspace Directory
+
+Open a Windows Command Prompt for VS
+Make new directory for Workspace:
+
+```
+$ cd /
+$ Mkdir FW
+$ cd FW
+$ Mkdir edk2-ws
+$ cd edk2-ws
+```
+
+---
+## Slide 13 Download the EDK II Source Code
+### Download the open source EDK II from GitHub
+
+Note:  if behind a firewall, set PROXYS FIRST (example shows for Intel Corp. -- Maybe different for your Corp. or GEO)
+```
+$ git config --global https.proxy proxy-dmz.intel.com:912
+$ git config --global http.proxy proxy-dmz.intel.com:911
+```
+
+From the command prompt use "git clone" to download following repos
+\#Edk2 - main core code
+```
+$ git clone -b Edk2Lab_22Q3 https://github.com/tianocore-training/edk2.git
+```
+\#EDK II "C" Library Repo
+```
+$ git clone https://github.com/tianocore/edk2-libc.git
+```
+\#EDK II Platforms Repo
+```
+$ git clone https:://github.com/tianocore/edk2-platforms.git
+```
+\#EDK II Non-OSI (Stand alone Binaries)
+```
+$ git clone https:://github.com/tianocore/edk2-non-osi.git
+```
+\#Intel FSP
+```
+$ git clone https://github.com/IntelFsp/FSP.git
```

Comment:
This should be :  $ git clone https://github.com/intel/FSP.git 

This was wrong in PPTx as well